scripts/makebigconflate

```diff
@@ -4,7 +4,7 @@
 
 HOOT_OPTS+= -D osm2ogr.ops=hoot::DecomposeBuildingRelationsVisitor -D conflate.add.score.tags=yes
 HOOT_OPTS+= -D hootapi.db.writer.overwrite.map=true -D hootapi.db.writer.create.user=true
-HOOT_OPTS+= -D hootapi.db.writer.email=test@test.com -D hootapi.db.reader.email=test@test.com
+HOOT_OPTS+= -D api.db..email=test@test.com
```

Comment:
good catch.  apparently that test doesn't care about that setting :-) but it does need to be fixed.